### DISCUSSION TITLE
How to compare a date with today in Handlebars?

### DISCUSSION THREAD

**Discussion by PzKfWg** [microsoft-search/pnp-modern-search#1611] [Q&A] [answered]
Hi,

I am trying to show an icon in Handlebars if a date managed property is older than today and another if not. I have a hard time understanding the correct syntax. I came up with:
```
{{#lt item.RefinableDate00 (moment)}}
	<div title="Rights expired / Droits expirés">
		<pnp-icon data-name="Warning" aria-hidden="true"></pnp-icon>
	</div>
{{/lt}}
{{#gt item.RefinableDate00 (moment)}}
	<div title="Rights in force / Droits en vigeur">
		<pnp-icon data-name="CheckMark" aria-hidden="true"></pnp-icon>
	</div>
{{/gt}}
```

But it doesn't work. Anytime a date exists, the first one returns true. Anyone knows how to compare a date with today in the Handlbars helper functions included in the modern search result web part? Thank you!

**Comment by wobba** [comment, marked as answer]
Something like this seems to work.... not sure if it's optimal.

```html
{{#lt (getDate item.Created 'X') (moment 'X')}}
Created: {{getDate item.Created "YYYY-MM-DD HH:MM"}}
<br>
Current: {{moment "YYYY-MM-DD HH:MM"}}
{{/lt}}
```

**Reply by PzKfWg** [reply]
Wow, it does work! I don't know what 'X' means, though. I understand that we end up comparing strings. Thanks a lot for that!

**Reply by wobba** [reply]
"X" means ticks - explained in the `moment` docs.
